Tests/UnitTests/Core/Utilities/QuickMathTests.cpp

```diff
@@ -20,42 +20,42 @@ const auto expDist = bdata::random(
 
 BOOST_AUTO_TEST_SUITE(Utilities)
 
-BOOST_DATA_TEST_CASE(fastInverseSqrt, expDist ^ bdata::xrange(100), exp, i) {
+BOOST_DATA_TEST_CASE(quickInverseSqrt, expDist ^ bdata::xrange(100), exp, i) {
   (void)i;
 
   const double x = std::pow(10, exp);
 
-  const float fastFloat = Acts::fastInverseSqrt(static_cast<float>(x));
-  const double fastDouble = Acts::fastInverseSqrt(x);
+  const float quickFloat = Acts::quickInverseSqrt(static_cast<float>(x));
+  const double quickDouble = Acts::quickInverseSqrt(x);
```

Comment:
Just saw that we have here as well 2 lines below

`const double stdFloat = 1.0 / std::sqrt(static_cast<float>(x));`
which should be `const float`

Reply:
nice catch!